analyzers/its/regression-test.ps1

```diff
@@ -130,8 +130,8 @@ function Initialize-OutputFolder() {
         Remove-Item -Recurse -Force output
     }
 
-    Write-Debug "Creating folder 'output\Issues'"
-    New-Item -ItemType directory -Path .\output\Issues | out-null
+    Write-Debug "Creating folder 'output'"
+    New-Item -ItemType directory -Path .\output | out-null
```

Comment:
Does the `output` needs to exist in advance, or can we delete this all together?

Reply:
This is needed - if one re-runs the tests output is deleted and re-created by these lines.
The logic can be moved in the code however (I'd make another PR for this.)

Reply:
It's fine to keep it here if it's really needed.